spec/spec_helper.rb

```diff
@@ -5,10 +5,12 @@
   add_filter 'test'
 end
 
-require_relative '../test/test_helper.rb'
+#require_relative '../test/test_helper.rb'
 
 require 'rspec'
 require 'pry'
+require 'policy_machine_test_app'
```

Comment:
It is a dead simple app.  I can push it to my repo and transfer it over when the paperwork clears, or I can ninja it into a branch of a private repo and then gitfu it when the paperwork clears.  Whacha think?